keiko-sql/src/main/kotlin/com/netflix/spinnaker/config/SqlQueueProperties.kt

```diff
@@ -0,0 +1,49 @@
+package com.netflix.spinnaker.config
+
+import com.netflix.spinnaker.kork.sql.config.SqlRetryProperties
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.validation.annotation.Validated
+import javax.validation.constraints.Pattern
+import javax.validation.constraints.Positive
+
+@ConfigurationProperties("keiko.queue.sql")
+@Validated
+class SqlQueueProperties {
+  /**
+   * Enables use of the SqlQueue implementation, disabling RedisQueue.
+   */
+  var enabled: Boolean = false
+  /**
+   * [queueName] namespaces the database tables so that multiple keiko queues can be collocated
+   * on the same database. When using a globally writeable data store with queue processors in
+   * multiple regions, it may be desirable to use the region as the [queueName]. Must match
+   * regexp "\w+".
```

Comment:
Use backticks on the regexp pattern. Kotlin docs use markdown: https://kotlinlang.org/docs/reference/kotlin-doc.html#inline-markup